wordinserter/renderers/com.py

```diff
@@ -595,6 +595,11 @@ def handle_format(self, op, parent_operation, element_range):
                 if op.text_align in alignment:
                     parent_operation.render.cell_object.Range.ParagraphFormat.Alignment = alignment[op.text_align]
 
+        if op.writing_mode:
+            orientations = {"vertical-lr": 1, "sideways-lr": 2}
+            if isinstance(parent_operation, TableCell):
+                parent_operation.render.cell_object.Range.Orientation = orientations[op.writing_mode]
```

Comment:
Can you ad a check to ensure that `op.writing_mode` is in `orientations`? Just to stop any unfortunate KeyErrors.